### DISCUSSION TITLE
not include sql.h

### DISCUSSION THREAD

**Discussion by hoangde2811** [mkleehammer/pyodbc#1182] [Q&A] [answered]
Dear developer!
When i use pyodide build to create the wheel file but it gives an error that there is no sql.h library in the pyodbc.h file.
Please help me to get it to work. 
I have attached a picture of the error below.
Sincerely thank you.
(https://user-images.githubusercontent.com/87251371/225799979-513602d8-8753-45e8-b3a2-3e8cd063c25d.png)

**Comment by gordthompson** [comment]
```
sudo apt install unixodbc-dev
```

will provide the missing file.

**Reply by v-chojas** [reply, marked as answer]
FYI: https://github.com/mkleehammer/pyodbc/issues?q=sql.h
